src/App.tsx

```diff
@@ -1,61 +1,140 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import {
+  Button,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  TextField,
+} from '@mui/material';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { User } from './type/User';
+import { Todos } from './type/Todos';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const todosWithUsers: Todos[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosWithUsers);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState('');
+  const [titleCorrect, setTitleCorrect] = useState(true);
+  const [userIdCorrect, setUserIdCorrect] = useState(true);
```

Comment:
```suggestion
  const [isTitleCorrect, setTitleCorrect] = useState(true);
  const [isUserIdCorrect, setUserIdCorrect] = useState(true);
```

or even better use isUserError and isTitleError